app/javascript/CommentSubscription/__tests__/CommentSubscription.test.jsx

```diff
@@ -54,7 +54,10 @@ describe('<CommentSubscription />', () => {
   it('should subscribe when the subscribe button is pressed', () => {
     const onSubscribe = jest.fn();
     const { getByText } = render(
-      <CommentSubscription onSubscribe={onSubscribe} />,
+      <CommentSubscription
+      onSubscribe={onSubscribe}
+      userStatus="logged-in"
+       />,
```

Comment:
nitpick (not blocking) worth double checking the formatting here - looks like it's went a little awry